vars/abapEnvironmentPullGitRepo.groovy

```diff
@@ -12,9 +12,9 @@ import java.util.UUID
 @Field def STEP_NAME = getClass().getName()
 @Field Set GENERAL_CONFIG_KEYS = [
     /**
-     * Specifies the host address of the SAP Cloud Platform ABAP Environment system
+     * Specifies the URL of the MANAGE_SOFTWARE_COMPONENT API of the communication scenario SAP_COM_0510 in the SAP Cloud Platform ABAP Environment system
      */
-    'host',
+    'url',
```

Comment:
to make it easier to re-use in the future shouldn't we rather go with something like

`...ServerUrl` and `...ApiEndpoint`

This would make it easier to re-use the server configuration in future scenarios

Reply:
As the ApiEndpoint is static, your proposal corresponds to the current behaviour on the master branch. 
'host' is similar to the 'ServerUrl' (except the 'https://').

When you setup the API on the ABAP system, you get the complete URL - hence the idea to change 'host' to 'url'. The user would just need to copy and paste it.

But, I think, you have a good point. In the future, several steps may use the same 'host'. 
Therefore, I currently tend to just close this PR.
In the end, the questions is: what is more convenient for the user?